package.json

```diff
@@ -7,7 +7,7 @@
     "format:check": "prettier --check \"**/*.md\""
   },
   "devDependencies": {
-    "prettier": "^2.6.1"
+    "prettier": "2.6.1"
```

Comment:
**suggestion:** I think we want this to be, don't we?

```suggestion
    "prettier": "^2.6.1"
```

For reference, this is a good tool to test semantic versioning for NPM (https://semver.npmjs.com/)